url.c

```diff
@@ -2269,19 +2270,30 @@ static int test_is_repo(url_t *url)
 
     if(
       url->is.mountable &&
-      (util_is_mountable(buf) || !util_check_exist(buf)) &&
+      (copy || util_is_mountable(buf) || !util_check_exist(buf)) &&
       !config.rescue &&
       (!config.download.instsys || util_check_exist(buf) == 'd')
     ) {
       if(!util_check_exist(buf) && opt) {
-        log_info("mount %s -> %s failed (ignored)\n", buf, sl->value);
+        log_info("%s %s -> %s failed (ignored)\n", copy ? "copy" : "mount", buf, sl->value);
       }
       else {
-        log_info("mount %s -> %s\n", buf, sl->value);
-
-        i = util_mount_ro(buf, sl->value, url->file_list) ? 0 : 1;
-        ok &= i;
-        if(!i) log_info("instsys mount failed: %s\n", sl->value);
+        if(copy) {
+          char *argv[3];
+          char *dst = strrchr(t, '/') ?: t;
+          log_info("copy %s -> %s\n", buf, dst);
+          argv[1] = buf;
```

Comment:
Right, but better be safe than sorry; leaving it with memory garbage is not a good idea IMHO. I'd at least initialize it with NULL to get a honest core dump rather than random results when passing memory garbage.